packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -61,7 +62,7 @@ const getUnionAndInterfaceTypes = () => {
 };
 
 module.exports = {
-    getMediaURL,
+    getStoreConfigData,
```

Comment:
Is there a way of avoiding this breaking change? Might be better to retain the `getMediaURL` and include the `getStoreConfigData`.

Reply:
I think we should think about https://github.com/magento/pwa-studio/issues/2120 if we need a breaking change also all developer will benefit oft this

Reply:
@davemacaulay Yes, getMediaURL has been restored

Reply:
@larsroettig Thanks for pointing that out